README.md

```diff
@@ -2,16 +2,30 @@
 
 [![Build Status](https://travis-ci.org/sonatype-nexus-community/nexus-repository-apt.svg?branch=master)](https://travis-ci.org/sonatype-nexus-community/nexus-repository-apt) [![DepShield Badge](https://depshield.sonatype.org/badges/sonatype-nexus-community/nexus-repository-apt/depshield.svg)](https://depshield.github.io)
 
+> **Huzzah!** APT is now part of Nexus Repository Manager. Version 3.17.0 includes the APT plugin by default.
+
+> **Filing issues:** If using 3.17.0 or later please file any issues at https://issues.sonatype.org/.
+
+> **Upgrading to 3.17.0:** If you are an existing APT user upgrading to 3.17 you will not be able to install the
+ community plugin. No other changes are required and your existing data will remain intact.
+
+>There are some differences from the community version of the plugin. First of all, we have added component records for 
+the .deb files which adds support for features such as *Search*, *Cleanup Policies*, *Tagging (PRO only)* and *Moving 
+between repositories (PRO only)*. We've also added support for *Restore Metadata Task*, *API for Repository creation 
+via Groovy* and we've written *help documentation*. The "retain N versions" feature has been removed as this conflicts 
```

Comment:
MInor: Rather than italic for help documentation, I think "normal" font but having a link would be better.  If you disagree about the link, I still (trivially) feel this way about the font...everything else italic is a feature, doc is not a feature.  To me it looks weird=)

Reply:
I'd like to get this PR merged ASAP. I'm assuming we can't link the docs until the docs are live? Maybe I'll just remove the italics and we can add the link later?

Reply:
Yep, it was minor anyway.  Go for the gold.